src/components/Dialogs/FittingDialog/FittingDialogComponent.tsx

```diff
@@ -33,6 +43,18 @@ export class FittingDialogComponent extends React.Component {
         return <SafeNumericInput value={isFinite(value) ? value : ""} placeholder={placeholder} onValueChange={onValueChange} buttonPosition="none" />;
     };
 
+    private exportResult = () => {
+        const content = AppStore.Instance.imageFittingStore.effectiveFrame?.fittingResult;
+        const fileName = `${AppStore.Instance.imageFittingStore.effectiveFrame?.filename}-${getTimestamp()}_2D-Fitting-Result`;
```

Comment:
Shall we unify the suffix to either "_XX-XXX-XXX" or "-XX_XXX_XXX"?

Reply:
It is now unified to "_XX-XXX-XXX"
![image](https://user-images.githubusercontent.com/106953609/193227471-7f39ae6b-2335-457b-83d2-7dff999f35fe.png)